website/src/pages/AdminBenchmarksPage/AdminBenchmarksPage.tsx

```diff
@@ -0,0 +1,85 @@
+import { useState } from "react";
+import DataForm from "../../common/DataForm";
+import Dropdown from "../../common/Dropdown";
+import useApiCall from "../../hooks/useApiCall";
+import ExecutionQueue from "./components/ExecutionQueue";
+import admin from "../../utils/admin";
+import { twMerge } from "tailwind-merge";
+
+const benchmarkWorkloads = [
+  "OLTP-SET",
+  "OLTP-READONLY",
+  "OLTP-READONLY-OLAP",
+  "TPCC",
+  "OLTP",
+];
```

Comment:
Can we make this list dynamic based on the list of benchmarks returned by the API?

Reply:
> Can we make this list dynamic based on the list of benchmarks returned by the API?

I thnk I know how to do this
```
func (s *Server) getBenchmarkNames(c *gin.Context) {
	c.JSON(http.StatusOK, gin.H{
		"benchmarks": s.benchmarkTypes,
	})
}
```

This code snippet if added in `go/server` and adding a respective route like `/api/get-benchmark-types`
Will give list which we can use in frontend.
@frouioui